packages/cf-component-loading/src/Loading.js

```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import Icon from 'cf-component-icon';
+import { Icon } from 'cf-component-icon';
+
+import { createComponent } from 'cf-style-container';
+
+const LoadingWrapper = createComponent(({}) => ({
+  width: 40,
+  animationDuration: '2s',
+  animationIterationCount: 'infinite',
+  animationTimingFunction: 'ease',
+  '>svg': {
+    height: '100%',
+    fill: 'transparent',
+    stroke: '#ff7900',
```

Comment:
I'm worried about our default `<Loading />` component (that will be used everywhere) being a 40px-wide orange circle. How does this look inside an orange `<Button />`? Have we audited where we currently use the spinner and made sure that this color works?